src/components/product/ProductGrid/ProductGrid.tsx

```diff
@@ -12,7 +13,7 @@ interface Props {
 function ProductGrid({ products, page, pageSize }: Props) {
   return (
     <ProductGridSkeleton loading={products.length === 0}>
-      <ul className="product-grid">
+      <ul className={styles.productGrid}>
```

Comment:
I noticed that in the other components we have the `fs` prefix but here is the only place we don't have it. 
Is there any reason for that?